### PR TITLE
Add Rimac automobili Icon

### PR DESCRIPTION
**Pull request by eshan25**
Issue: #6149
Alexa rank: 115,924

Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

Description
homepage: https://www.rimac-automobili.com/
hex: FFFFFF
SVG from: https://www.rimac-automobili.com/
Optimize: https://jakearchibald.github.io/svgomg/

### CODE REVIEW

File: icons/rimac.svg
Comment (ericcornelissen) [inline icons/rimac.svg:1]: Before we review this, why did you choose to add this as the logo rather than just the "R"?

File: _data/simple-icons.json
Comment (ericcornelissen) [inline _data/simple-icons.json:7713]: Why did you go with `#FFFFFF`? From the website I would maybe expect a color like `#0A222E` :thinking:

File: _data/simple-icons.json
Comment (ericcornelissen) [inline _data/simple-icons.json:7712]: While I agree "Rimac" is the better option, I wonder whether this should be "Rimac automobili" :thinking: What do you think?

File: _data/simple-icons.json
Comment (ericcornelissen) [inline _data/simple-icons.json:7714]: I think we should set this to the media center rather than the homepage. It also has embedded SVGs plus various versions of the logo in PDFs. Also, I think the media center has sufficient resources to consider it for [the guidelines entry](https://github.com/simple-icons/simple-icons/blob/b12222e498909099e22ba1c51c21fda544c3bae4/CONTRIBUTING.md#optional-data).

```suggestion
            "source": "https://www.rimac-automobili.com/media/",
            "guidelines": "https://www.rimac-automobili.com/media/"
```

**Review by ericcornelissen — CHANGES_REQUESTED** [review]
Thank you for taking the time to contribute @eshan25 :smiley: Before we can do a full review please have a look at the questions below.

PS. there's no need to keep your Pull Request in sync with `develop` constantly. Once your Pull Request has been approved one of the maintainers will take care of merging it in.

**Comment by eshan25** [comment]
> Thank you for taking the time to contribute @eshan25 😃 Before we can do a full review please have a look at the questions below.
> 
> PS. there's no need to keep your Pull Request in sync with `develop` constantly. Once your Pull Request has been approved one of the maintainers will take care of merging it in.

Hi. Im sorry, Im new to GitHub. I will take note on this and wont sync it anymore.

File: icons/rimac.svg
Comment (eshan25) [reply icons/rimac.svg:1]: Hi. Since the logo provided in the pdf file downloaded from the media center page included the "RIMAC AUTOMOBILI", I decided to add it too inside the svg. I can amend it by removing the wordings, leaving the "R".

File: _data/simple-icons.json
Comment (eshan25) [reply _data/simple-icons.json:7713]: Hi. I assumed the color would follow as the logo in the main page, therefore I put it as #FFFFFF. I will amend this to #0A222E.

File: _data/simple-icons.json
Comment (eshan25) [reply _data/simple-icons.json:7712]: Hi. Yes, in my opinion, Rimac automobili should be appropriate since it is the full brand name. As above, will amend this.

File: _data/simple-icons.json
Comment (eshan25) [reply _data/simple-icons.json:7714]: Hi. I will change this accordingly and sorry for many mistakes done. I will amend ASAP. Thank you for your time.

File: icons/rimac.svg
Comment (ericcornelissen) [reply icons/rimac.svg:1]: I'm not that familiar with the brand so you might be better to judge this, but there are various places where they use just the "R" and I didn't spot anything saying we can't use that :thinking:

File: _data/simple-icons.json
Comment (ericcornelissen) [reply _data/simple-icons.json]: No problem, it's not a mistake :slightly_smiling_face:

File: icons/rimac.svg
Comment (eshan25) [reply icons/rimac.svg:1]: Will amend it to "R" only later tonight or as soon as possible... Thanks again for your time to review this...

Comment (eshan25) [reply icons/rimac.svg]: Hi. I have made the amendments. Do tell if anything needs to be updated again. Thanks for reviewing.

**Review by ericcornelissen — APPROVED** [review]
Awesome work @eshan25! The SVG looks good and I agree with the metadata as is :+1:

**Comment by ericcornelissen** [comment]
Thank you or the contribution @eshan25, it is much appreciated :smile:

**Pull request merged**
